Cargo.toml

```diff
@@ -1,16 +1,16 @@
 # Copyright (c) 2023 MASSA LABS <info@massa.net>
 
 [package]
-name = "massa_proto_bindings"
+name = "massa_proto_rs"
 version = "0.1.0"
 edition = "2021"
 description = "Protobuf rust bindings for the Massa blockchain"
-repository = "https://github.com/massalabs/massa-proto-bindings/"
+repository = "https://github.com/massalabs/massa-proto/"
```

Comment:
missing a `-rs` at the end of the url, isn't it?

Reply:
Good catch